src/components/Board.js

```diff
@@ -8,18 +8,18 @@ import Card from "./Card";
 // import CARD_DATA from "../data/card-data.json";
 
 const Board = (props) => {
-  const API_URl_BASE = `${props.url}/${props.boardName}`;
```

Comment:
Took this out because when you delete, the base url is totally different. 
The base url = https://inspiration-board.herokuapp.com
The URL to get boards = https://inspiration-board.herokuapp.com/boards
The URL to delete cards goes straight to cards: https://inspiration-board.herokuapp.com/cards